packages/contentful--app-scripts/lib/clean-up/clean-up-bundles.js

```diff
@@ -0,0 +1,60 @@
+const chalk = require('chalk');
+const ora = require('ora');
+const { throwError } = require('../utils');
+const { createClient } = require('contentful-management');
+
+async function deleteBundle(bundleId, index, maxIndex, client, settings) {
+  const deleteSpinner = ora(`Deleting ${index + 1} out of ${maxIndex} bundles...`).start();
+  await client.appBundle.delete({
+    appBundleId: bundleId,
+    appDefinitionId: settings.definition.value,
+    organizationId: settings.organization.value,
+  });
+  deleteSpinner.stop();
+  console.log(`${chalk.green('Done:')} ${index + 1}/${maxIndex} bundles deleted successfully`);
+}
+
+async function cleanUpBundles(settings) {
+  let bundles;
+  const client = createClient({ accessToken: settings.accessToken }, { type: 'plain' });
+  const bundlesSpinner = ora(`Fetching all bundles...`).start();
+  try {
+    bundles = await client.appBundle.getMany({
+      appDefinitionId: settings.definition.value,
+      organizationId: settings.organization.value,
+    });
+  } catch (e) {
+    throwError(e, 'Something went wrong fetching the bundles');
+  }
+
+  bundlesSpinner.stop();
+
+  const bundlesToDelete = bundles.items.slice(settings.keep);
+
+  if (bundlesToDelete.length < 1) {
+    console.log(`${chalk.yellow('Warning:')} There is nothing to delete`);
+  }
+
+  console.log(`
+
+${chalk.cyan('Info:')} ${bundlesToDelete.length} bundle${
+    bundlesToDelete.length > 1 ? 's' : ''
+  } will be deleted
+
+  `);
+
+  try {
+    for (const [index, bundle] of bundlesToDelete.entries()) {
+      await deleteBundle(bundle.sys.id, index, bundlesToDelete.length, client, settings);
```

Comment:
Shall we maybe parallelize and batch this? Not pushing for this specific library, but I was thinking something like [this](https://caolan.github.io/async/v3/docs.html#parallelLimit) to speed up the process and not get 429s